proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+# OCI Runtime-Tools and Image-Tools Project Proposals
+
+## Abstract
+The below proposal contemplates creation of two tools projects (runtime-tools and image-tools.)
+These would be associated with the OCI specification projects (runtime-spec and image-spec) and serve as repositories for testing  tools. 
+
+## Background
+Testing tools have been organically developed by the OCI developer community.
+These include:
+* Runtime-spec test code in a OCITools repository
+* Image-spec test code in the image-spec repository
+
+On Monday, August 22nd, 2016, the OCI Certification Program WG designated Chris Aniszczyk and Rob Dolin to draft a proposal for formalizing approval of tools.
+
+On Wednesday, August 24th, 2016, the OCI Developer Community (TDC) met and recommended establishing a tools repository corresponding to each of the spec repositories.  
+
+## Proposal
+With repositories under the http://github.com/OpenContainers organization:
+* Rename the OCITools repository as runtime-tools
+* Create a new repository named image-tools and populate this with the files from: https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool
+
+### Project Descriptions
+Below are brief draft project descriptions intended to serve as an initial abstract for each project:
+* Runtime-Tools: Tools for testing of container runtimes implementing the OCI runtime-spec.  
+This includes code that tests a runtime's conformance to the OCI Container Runtime spec.
+* Image-Tools: Tools for testing of container images implementing the OCI image-spec.  
+This includes code that validates a file's conformance to the OCI Container Image Format spec.  
+
+### Initial Maintainers
```

Comment:
@crosbymichael This is a good question about initial maintainers (and how to add/remove maintainers.)  I've seen organizations do things multiple ways with the three most common being:

- Project proposal (or org charter) lists initial leadership (IIRC, OCI did this with runc maintainers and runtime-spec maintainers)
- Election for initial leadership (IIRC, OCI did this with TOB membership and the image-spec maintainers)
- Pay to be in leadership (I've seen orgs with membership pricing tiers do this)

I thought making the specs repo maintainers the initial maintainers of the tools repos and adding the next three active contributors seemed reasonable.  

(I also assume that anyone can opt-out of being a maintainer at any time.)

I would like to see OCI clarify how maintainers are added and removed after project creation since the OCI charter is a bit vague on this, but that's probably an issue for another pull request.